otter/test/convergence/test_service.py

```diff
@@ -632,7 +632,7 @@ def test_returns_retry(self):
         def plan(*args, **kwargs):
             return pbag([
                 TestStep(Effect(Constant((StepResult.SUCCESS, [])))),
-                ConvergeLater(),
+                ConvergeLater(reasons=freeze(['mywish'])),
```

Comment:
> Non-blocking - are reasons supposed to be a pyrsistent data structure?

The need to be prysistent DS so that they are hashable and usable in tests. 

> If so, should that be in the ConvergeLater attributes constructor?

I tried putting `Attribute('reasons', instance_of=PVector)` in constructor but `PVector` conflicted with some other class and I didn't want to spend time figuring out how to fix it.